src/public-ludumdare.com/main-ld.js

```diff
@@ -1,102 +1,113 @@
-import DialogActivate					from 'com/dialog-activate/activate';
-import DialogReset						from 'com/dialog-reset/reset';
-import DialogPassword					from 'com/dialog-password/password';
-import DialogAuth						from 'com/dialog-auth/auth';
-import DialogSession					from 'com/dialog-session/session';
-import DialogSavebug					from 'com/dialog-savebug/savebug';
-import DialogUserConfirm				from 'com/dialog-user/user-confirm';
-
-import DialogSubmit						from 'com/dialog-submit/submit';
-import DialogTV							from 'com/dialog-tv/tv';
-
-import DialogCreate						from 'com/dialog-create/create';
-
-//import AlertBase						from 'com/alert-base/base';
-
-import $Node							from '../shrub/js/node/node';
-import $User							from '../shrub/js/user/user';
-import $NodeLove						from '../shrub/js/node/node_love';
+import {h, render, Component, options} from 'preact/preact';
+import {initDevTools} from 'preact-devtools/devtools';
+import Sanitize from '../internal/sanitize/sanitize';
+import NavSpinner from 'com/nav-spinner/spinner';
+
+import ViewBar from 'com/view-bar/bar';
+import ViewHeader from 'com/view-header/header';
+import ViewSidebar from 'com/view-sidebar/sidebar';
+import ViewContent from 'com/view-content/content';
+import ViewFooter from 'com/view-footer/footer';
+import ViewHome from 'com/view-home/home';
+
+import Router from 'com/router/router';
+import Route from 'com/router/route';
+
+import PageHome from 'com/pages/home/home';
+import PagePage from 'com/pages/page/page';
+import PagePost from 'com/pages/post/post';
+import PageItem from 'com/pages/item/item';
+// import PageGames from 'com/pages/games/games'; //TODO: FIX
+import PageTag from 'com/pages/tag/tag';
+import PageUser from 'com/pages/user/user';
+import PageUsers from 'com/pages/users/users';
+import PageEvent from 'com/pages/event/event';
+import PageEvents from 'com/pages/events/events';
+import PageError from 'com/pages/error/error';
+
+import DialogUnfinished from 'com/dialog-unfinished/unfinished';
+import DialogLogin from 'com/dialog-login/login';
+import DialogRegister from 'com/dialog-register/register';
+import DialogActivate from 'com/dialog-activate/activate';
+import DialogReset from 'com/dialog-reset/reset';
+import DialogPassword from 'com/dialog-password/password';
+import DialogAuth from 'com/dialog-auth/auth';
+import DialogSession from 'com/dialog-session/session';
+import DialogSavebug from 'com/dialog-savebug/savebug';
+import DialogUserConfirm from 'com/dialog-user/user-confirm';
+
+import DialogSubmit from 'com/dialog-submit/submit';
+import DialogTV from 'com/dialog-tv/tv';
+
+import DialogCreate from 'com/dialog-create/create';
+
+//import AlertBase                        from 'com/alert-base/base';
+
+import $Node from '../shrub/js/node/node';
+import $User from '../shrub/js/user/user';
+import $NodeLove from '../shrub/js/node/node_love';
 
 window.LUDUMDARE_ROOT = '/';
 window.SITE_ROOT = 1;
 
 if ( SITE_DEBUG ) {
-	initDevTools();
+    initDevTools();
```

Comment:
And it looks like the prefix tabs got converted to 4 spaces.

Reply:
weird, they're a tab in my IDE.